docs/src/main/tut/entity.md

```diff
@@ -12,8 +12,8 @@ streaming), but before the body has been fully received. The same applies to the
 http client usage, where you can start a connection before the body is fully
 materialized. You don't have to load the full body into memory to submit the
 request either. Taking a look at `Request[F]` and `Response[F]`, both have a body of
-type `EntityBody[F]`, which is simply an alias to `Stream[F, ByteVector]`. To
-understand `Stream`, take a look at the [streams-tutorial].
+type `EntityBody[F]`, which is simply an alias to `Stream[F, Byte]`. To
+understand `Stream`, take a look at the [streams-tutorial] or a youtube video [introduction-to-functional-streams].
```

Comment:
Don't think _a youtube video_ is necessary could probably remove so it reads _or introduction-to-functional-streams_

Reply:
@ChristopherDavenport 👍